app/src/main/java/org/sopt/study/mvvmstudy/MainActivity.kt

```diff
@@ -0,0 +1,58 @@
+package org.sopt.study.mvvmstudy
+
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import androidx.appcompat.app.AlertDialog
+import androidx.lifecycle.Observer
+import androidx.lifecycle.ViewModelProviders
+import org.sopt.study.mvvmstudy.data.Contact
+import org.sopt.study.mvvmstudy.data.ContactDatabase
+import org.sopt.study.mvvmstudy.data.ContactViewModel
+import org.sopt.study.mvvmstudy.databinding.ActivityMainBinding
+
+class MainActivity : AppCompatActivity() {
+
+    private lateinit var contactViewModel: ContactViewModel
+    private lateinit var binding : ActivityMainBinding
+    private lateinit var contactAdapter : ContactAdapter
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = ActivityMainBinding.inflate(layoutInflater)
+
+        initAdapter()
+
+        contactViewModel = ViewModelProviders.of(this).get(ContactViewModel::class.java)
+        contactViewModel.getAll().observe(this, Observer<List<Contact>>{contacts->contactAdapter.setContacts(contacts!!)})
```

Comment:
```suggestion
        contactViewModel.getAll().observe(this) {contacts->contactAdapter.setContacts(contacts!!)}
```
이것 또한 넘나 자바식 코드라서.. 코틀린에서는 이런식으로 간결하게 쓸 수 있어용~!!